Porting/vote_admin_guide.pod

```diff
@@ -172,11 +172,11 @@ edit the Perl Core mailing list admins to match the incoming Steering Council
 
 =item *
 
-update the GitHub "steering" team to match incoming Steering Council
+update the L<https://github.com/orgs/Perl/teams/perl-steering-council/members|GitHub "steering" team> to match incoming Steering Council
```

Comment:
The parts need swapping; this needs to be written as

```
L<GitHub "steering" team|https://...>
```

Reply:
changed in c99b1f8fa7386d607e3b26fda3ea413d8e7484f1